package.json

```diff
@@ -26,7 +24,7 @@
     "analyze:server": "BUNDLE_ANALYZE=server npm run build",
     "analyze:client": "BUNDLE_ANALYZE=browser npm run build",
     "fetch-data": "npm run copy-static && npm run update-dict && npm run update-team",
-    "postinstall": "patch-package && npm run fetch-data",
+    "postinstall": "git submodule update --init && npm i --prefix styles && npm run fetch-data",
```

Comment:
so are we installing all dependencies here or only production dependencies?

Reply:
good point, imho production should be enough